modules/grpc/otel/filterx/otel-logrecord.cpp

```diff
@@ -62,15 +120,15 @@ OtelLogRecordCpp::Marshal(void)
 
 bool OtelLogRecordCpp::SetField(const gchar *attribute, FilterXObject *value)
 {
-  // TODO
-  return false;
+  std::string fieldName(attribute);
+  return otel_converter.Set(&this->logRecord, fieldName, value);
```

Comment:
[nitpick]

IIRC you can just pass `attribute` here as the 2nd param, and a new `std::string` will be created.
Same in `GetField()`.